Library/Formula/lnav.rb

```diff
@@ -21,6 +21,7 @@ class Lnav < Formula
 
   depends_on "readline"
   depends_on "pcre"
+  depends_on "curl"
```

Comment:
You are correct, I believe, it doesn't. Our standard curl doesn't either though. You'd have to specify `curl => "with-libssh2`.

Unless it's feature critical I'd rather make it optional though and let people invoke it as desired.